### DISCUSSION TITLE
WYSIWYG (Visual) Editor: Font sizes do not render as expected in the wiki

### DISCUSSION THREAD

**Discussion by siddhartha-lal** [requarks/wiki#3668] [Error / Bug Report]
**Describe the bug**
Different font sizes do not render as expected in the final wiki. All text in a table or a normal paragraph, irrespective of its size in the editor, is rendered in the default size. **The text looks as expected in the visual editor itself.**

**To Reproduce**
Steps to reproduce the behaviour:
1. Create a table or a paragraph
2. Write some text in the table/paragraph and choose a size other than default
3. Visit the page created in Step 2

**Expected behaviour**
Different font sizes should be rendered differently.

**Screenshots**
![editor_2](https://user-images.githubusercontent.com/56102539/80302489-6c692980-87d0-11ea-8b9c-654b9473702f.png)
![editor_1](https://user-images.githubusercontent.com/56102539/80302491-6d9a5680-87d0-11ea-9bd1-70b809004bd7.png)

**Host Info (please complete the following information):**
 - OS: Ubuntu 18.04
 - Wiki.js version: 2.3.77
 - Database engine: MySQL 5.7.29

**Additional context**
None. Dunno why someone else has not already pointed it out.

**Comment by d1eselboy** [comment]
We have the same issue

**Comment by seanecoffey** [comment]
Also having this one.  This is just a CSS issue but there is a workaround. 
The WYSIWYG editor applies the `text-big` class (or tiny etc.) to whatever you apply, but I think this is only applicable when nested in the .ckeditor container.

You can add the classes to the page, under:
Edit Page > Page > Styles

`.text-big {
  font-size: 1.4em
}`

Or whatever class you need to add.  Inspector can show you the font-sizes in the ckeditor window to apply.
